protocol/contracts/profit/src/lib.rs

```diff
@@ -6,20 +6,15 @@ pub mod typedefs;
 #[cfg(feature = "stub")]
 pub mod stub;
 
-#[cfg(any(feature = "astroport", feature = "osmosis"))]
+#[cfg(all(feature = "contract", any(feature = "astroport", feature = "osmosis")))]
 mod access_control;
-
-#[cfg(any(feature = "astroport", feature = "osmosis"))]
+#[cfg(all(feature = "contract", any(feature = "astroport", feature = "osmosis")))]
 pub mod contract;
-
 #[cfg(any(feature = "astroport", feature = "osmosis"))]
 pub mod error;
-
-#[cfg(any(feature = "astroport", feature = "osmosis"))]
+#[cfg(all(feature = "contract", any(feature = "astroport", feature = "osmosis")))]
 pub mod profit;
-
 #[cfg(any(feature = "astroport", feature = "osmosis"))]
 pub mod result;
-
-#[cfg(any(feature = "astroport", feature = "osmosis"))]
+#[cfg(all(feature = "contract", any(feature = "astroport", feature = "osmosis")))]
```

Comment:
I guess adding `contract` as a dependent feature to `astroport` and `osmosis` would turn this change into useless.

Reply:
Not necessarily. `error` module doesn't require it.